pkg/gui/gui.go

```diff
@@ -778,3 +784,18 @@ func (gui *Gui) OnUIThread(f func() error) {
 		return f()
 	})
 }
+
+func (gui *Gui) UpdateWindowTitle() error {
```

Comment:
Could we put this function in `pkg/commands/oscommands/os.go`? We can then invoke it as `gui.OSCommand.UpdateWindowTitle()`.

Reply:
Absolutely, I wasn't at all confident about that location